tests/testthat/test-parsers.R

```diff
@@ -0,0 +1,25 @@
+test_that("parse_errors_to_df parses correct assertion type and correct error_df", {
+  data <- data.frame(
+    V1 = c(1, 0),
+    V2 = c(-1, -2)
+  )
```

Comment:
Please modify the test case so that not all values in the tested column fail the validation (e.g. `V1 = c(1, 0, -3)`).

Reply:
Test case modified in [33b781a](https://github.com/Appsilon/data.validator/pull/59/commits/33b781a8bec60c537c97cc06b36b88657146e67e).